### ISSUE TITLE
Font-Awesome Icons should use aria-hidden="true"

### ISSUE THREAD

**Issue by Skrypt** [StartBootstrap/startbootstrap-coming-soon#57]
**aria-hidden="true"** should be used on font-awesome icons so that it be proper HTML semantic.
Same goes for CleanBlog and Agency themes that we use in Orchard Core. Using SonarQube to analyze our code relevated this issue.

Example : 

`<i class="fas fa-circle" aria-hidden="true"></i>`
